lib/models/Cat.js

```diff
@@ -0,0 +1,27 @@
+const pool = require('../utils/pool');
+module.exports = class Cat {
+  id;
+  name;
+ 
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.url = row.url;
+    this.year = row.year;
+    this.lives = row.lives;
+    this.isSidekick = row.is_sidekick;
```

Comment:
nice work recognizing the difference of a key in an object vs. a column in the db tables.